mobile/src/lib/network.js

```diff
@@ -0,0 +1,12 @@
+export const watchNetworkState = (onConnectionChange) => {
+  document.addEventListener('offline', onConnectionChange, false)
+  document.addEventListener('online', onConnectionChange, false)
+}
+
+export const hasCordovaPlugin = window.navigator.connection !== undefined && window.Connection !== undefined
+
+export const getConnectionType = () => hasCordovaPlugin ? navigator.connection.type : 'wifi'
+
+export const onWifi = connection => connection === 'wifi'
```

Comment:
`Connection.WIFI` exists only if you have cordova plugins loaded. So, in a standalone web version (as I use in development), we don't have the constant `Connection.WIFI`.

But it may be a good think to change this string into a built-in constant.